data.json

```diff
@@ -914,7 +914,7 @@
   {
     "name": "aminos",
     "githubUsername": "amino19",
-    "imageUrl": "https://user-images.githubusercontent.com/75872316/118384765-6dad0480-b626-11eb-85ce-a6af24c19181.png",
+    "imageUrl": "https://user-images.githubusercontent.com/75872316/125152497-91924f80-e16a-11eb-8366-8d5c3d26df10.jpg",
```

Comment:
I'm guessing the filename here should be `.png` for the Prettier to stop failing?

Reply:
@Panquesito7 : Yes actually I also thought that previously, but some members said it occurs sometimes.
Also, there're more `.jpg` image inside!